Update philips_js.markdown

TonyPeponi opened the pull request:
I've used the information from:
https://github.com/eslavnov/pylips/blob/master/docs/Home.md
to update the endpoints for the api version check.

<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).

  Before submitting your pull request, please verify that you have chosen the correct target branch,
  and that the PR preview looks fine and does not include unrelated changes.
-->
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->

The endpoint to check the api version on Android Philips TVs was missing.
    I added it based on https://github.com/eslavnov/pylips/blob/master/docs/Home.md
    and my experience with an Android Philips TV


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **Documentation**
	- Enhanced integration instructions for Philips TVs.
	- Clarified prerequisites for compatibility verification before adding the integration.
	- Reformatted instructions for checking the TV's response into a clearer, step-by-step format.
	- Adjusted the presentation of prerequisites to a bullet point format for improved readability.
	- Revised phrasing in instructions for older TVs to consolidate protocol information for clarity.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Commit:
Update philips_js.markdown

I've used the information from:
https://github.com/eslavnov/pylips/blob/master/docs/Home.md

to update the endpoints for the api version check.

c0ffeeca7 commented on source/_integrations/philips_js.markdown:
```diff
@@ -24,7 +24,15 @@ ha_integration_type: integration
 The `philips_js` platform allows you to control Philips TVs which expose the [jointSPACE](http://jointspace.sourceforge.net/) JSON-API.
 
-If your TV responds to `http://IP_ADDRESS_OF_TV:1925/system` then this integration can be used. In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).
+**Differences between Android TVs and non-Android TVs**
+1) Android TVs use `https://`, non-Android TV use `http://`
+2) Android TVs primarily use port 1926, non-Android TV use port 1925 only
```
```suggestion

**Differences between Android TVs and non-Android TVs**: 

1. Android TVs use `https://`, non-Android TV use `http://`.
2. Android TVs primarily use port 1926, non-Android TVs use port 1925 only.
```

c0ffeeca7 commented on source/_integrations/philips_js.markdown:
```diff
@@ -24,7 +24,15 @@ ha_integration_type: integration
+1) Android TVs use `https://`, non-Android TV use `http://`
+2) Android TVs primarily use port 1926, non-Android TV use port 1925 only
+   
+For non-Android TV check if your TV responds to `http://IP_ADDRESS_OF_TV:1925/system`. 
```
```suggestion
If you have a non-Android TV, check if your TV responds to `http://IP_ADDRESS_OF_TV:1925/system`. 
```

c0ffeeca7 commented on source/_integrations/philips_js.markdown:
```diff
@@ -24,7 +24,15 @@ ha_integration_type: integration
+   
+For non-Android TV check if your TV responds to `http://IP_ADDRESS_OF_TV:1925/system`. 
+
+For Android TV check if your TV responds to `https://IP_ADDRESS_OF_TV:1926/system`.
```
```suggestion
If you have an Android TV, check if your TV responds to `https://IP_ADDRESS_OF_TV:1926/system`.
```

c0ffeeca7 commented on source/_integrations/philips_js.markdown:
```diff
@@ -24,7 +24,15 @@ ha_integration_type: integration
+
+For Android TV check if your TV responds to `https://IP_ADDRESS_OF_TV:1926/system`.
+
+If you get a response then this integration can be used. In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).
```
```suggestion
If you get a response, then this integration can be used. In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).
```

c0ffeeca7 committed:
tiny tweaks

elupus commented on source/_integrations/philips_js.markdown:
```diff
@@ -24,7 +24,17 @@ ha_integration_type: integration
+**Differences between Android TVs and non-Android TVs**: 
+
+1. Android TVs use `https://`, non-Android TV use `http://`.
+2. Android TVs primarily use port 1926, non-Android TVs use port 1925 only.
```
This is incorrect as far as i know..Titan TVs use 1926 too i think

TonyPeponi replied:
Is it Titan TV a Philips model?
I updated the Philips TV documentation based on my experience http://xx.xx.xx.xx:1925/system was not working for my android Philips TV. I found a solution and shared it in the docs.

elupus replied:
It is the OS on the tv. Android, Saphi, titan and some unknown legacy OS.

Some use https (1926). As far as i know they all respons on 1925 for system info. But some can only be controlled on 1926 after auth.

So keep it generic.

TonyPeponi replied:
Thanks, I made it generic.

Commit:
Update philips_js.markdown

Use a Proper Heading Instead of Emphasized Text
Instead of using bold text for the section title

Commit:
Update philips_js.markdown

Made it generic, removed the Android, non-Android referneces

Commit:
Update philips_js.markdown

fixed misspelling, removed trailing white spaces

c0ffeeca7 commented on source/_integrations/philips_js.markdown:
```diff
@@ -24,7 +24,21 @@ ha_integration_type: integration
 The `philips_js` platform allows you to control Philips TVs which expose the [jointSPACE](http://jointspace.sourceforge.net/) JSON-API.
 
-If your TV responds to `http://IP_ADDRESS_OF_TV:1925/system` then this integration can be used. In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).
+
+## Differences between TVs 
+
+1. Some TVs use `https://`, other TVs use `http://`.
+2. Some TVs primarily use port 1926, other TVs use port 1925 only.
+
+## Check if this integration can be used
+Check if your TV responds to one of the following combinations:
+
+- `http://IP_ADDRESS_OF_TV:1925/system`
+- `https://IP_ADDRESS_OF_TV:1925/system`
+- `http://IP_ADDRESS_OF_TV:1926/system`
+- `https://IP_ADDRESS_OF_TV:1926/system`
+
+If you get a response, then this integration can be used. In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).
```
```suggestion

## Prerequisites

Before adding the integration, it is worth checking if this integration works for you. As there are differences between TVs, you may need to try different addresses:

- Some TVs use `https://`, other TVs use `http://`.
- Some TVs primarily use port 1926, other TVs use port 1925 only.

### Check if this integration can be used

1. Check if your TV responds to one of the following combinations:
    - `http://IP_ADDRESS_OF_TV:1925/system`
    - `https://IP_ADDRESS_OF_TV:1925/system`
    - `http://IP_ADDRESS_OF_TV:1926/system`
    - `https://IP_ADDRESS_OF_TV:1926/system`
2. If you get a response, then this integration can be used. 
   - In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).
```

c0ffeeca7 committed:
convert to prereq section

elupus commented on source/_integrations/philips_js.markdown:
```diff
@@ -24,7 +24,23 @@ ha_integration_type: integration
+1. Check if your TV responds to one of the following combinations:
+    - `http://IP_ADDRESS_OF_TV:1925/system`
+    - `https://IP_ADDRESS_OF_TV:1925/system`
+    - `http://IP_ADDRESS_OF_TV:1926/system`
```
```suggestion
```

elupus commented on source/_integrations/philips_js.markdown:
```diff
@@ -24,7 +24,23 @@ ha_integration_type: integration
+
+1. Check if your TV responds to one of the following combinations:
+    - `http://IP_ADDRESS_OF_TV:1925/system`
+    - `https://IP_ADDRESS_OF_TV:1925/system`
```
```suggestion
```

elupus reviewed (commented):
1925 is always http
1926 is always https

c0ffeeca7 commented on source/_integrations/philips_js.markdown:
```diff
@@ -24,7 +24,23 @@ ha_integration_type: integration
+    - `https://IP_ADDRESS_OF_TV:1926/system`
+2. If you get a response, then this integration can be used. 
+   - In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).
 For older TVs follow instructions on how to activate the API and if your model is supported [here](http://jointspace.sourceforge.net/download.html). Note that not all listed, jointSPACE-enabled devices will have JSON-interface running on port 1925. This is true at least for some models before year 2011.
```
```suggestion
3. For older TVs follow instructions on how to activate the API and if your model is supported [here](http://jointspace.sourceforge.net/download.html). 
   - Note that not all listed, jointSPACE-enabled devices will have JSON-interface running on port 1925. 
   - This is true at least for some models before year 2011.
```

c0ffeeca7 commented on source/_integrations/philips_js.markdown:
```diff
@@ -24,7 +24,23 @@ ha_integration_type: integration
+2. If you get a response, then this integration can be used. 
+   - In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).
 For older TVs follow instructions on how to activate the API and if your model is supported [here](http://jointspace.sourceforge.net/download.html). Note that not all listed, jointSPACE-enabled devices will have JSON-interface running on port 1925. This is true at least for some models before year 2011.
 
 Also, note that version 6 of the API needs to be authenticated by a PIN code displayed on your TV.
```
```suggestion
    - Also, note that version 6 of the API needs to be authenticated by a PIN code displayed on your TV.
```

c0ffeeca7 committed:
Apply suggestions from code review

c0ffeeca7 commented on source/_integrations/philips_js.markdown:
```diff
@@ -24,10 +24,27 @@ ha_integration_type: integration
+3. For older TVs follow instructions on how to activate the API and if your model is supported [here](http://jointspace.sourceforge.net/download.html). 
+   - Note that not all listed, jointSPACE-enabled devices will have JSON-interface running on port 1925. 
+   - This is true at least for some models before year 2011.
+    - Also, note that version 6 of the API needs to be authenticated by a PIN code displayed on your TV.
```
```suggestion
   - Also, note that version 6 of the API needs to be authenticated by a PIN code displayed on your TV.
```

c0ffeeca7 committed:
Update source/_integrations/philips_js.markdown

c0ffeeca7 commented:
I rephrased it a bit to move into a Prerequisite section that we usually use to list things that need to be checked/done before adding the integration. Let me know if the rephrased version is still correct or not.

c0ffeeca7 committed:
fix indents

Co-authored-by: Joakim Plate <elupus@ecce.se>

c0ffeeca7 commented on source/_integrations/philips_js.markdown:
```diff
@@ -24,10 +24,25 @@ ha_integration_type: integration
+    - `https://IP_ADDRESS_OF_TV:1926/system`
+2. If you get a response, then this integration can be used. 
+   - In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).
+3. For older TVs follow instructions on how to activate the API and if your model is supported [here](http://jointspace.sourceforge.net/download.html). 
```
```suggestion
3. For older TVs, follow instructions on how to activate the API and if your model is supported [here](http://jointspace.sourceforge.net/download.html). 
```

c0ffeeca7 committed:
typo

c0ffeeca7 commented on source/_integrations/philips_js.markdown:
```diff
@@ -24,10 +24,25 @@ ha_integration_type: integration
+
+Before adding the integration, it is worth checking if this integration works for you. As there are differences between TVs, you may need to try different addresses:
+
+- Some TVs use `https://`, other TVs use `http://`.
+- Some TVs primarily use port 1926, other TVs use port 1925 only.
```
```suggestion
- Some TVs use `https://` and port 1926, other TVs use `http://` and port 1925.
```

c0ffeeca7 committed:
implement review feedback

c0ffeeca7 commented:
Thank you, @TonyPeponi and @elupus :+1:
